README.md

```diff
@@ -1,7 +1,176 @@
-# microsoft-teams-test-tab
+This documents outlines the steps required to add your changes to Yuri's app and run the app in your local environment.
```

Comment:
I would just say the "Microsoft Teams Test App" instead of "Yuri's app". 😁

Reply:
Sure, will change that.